src/strings-in-block/class-compose.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace WPML\PB\Gutenberg\StringsInBlock;
+
+class Compose implements StringsInBlock {
```

Comment:
I suggested using "Compose" because:

> That's the purpose of the composite pattern after all: to compose different entities into something which can be handled as a whole.

It still does not convince me, as I don't think the method should tell the reader that "I'm implementing the Composite pattern".

In general, I'm not a big fan of class names telling you I'm implementing a specific pattern: the class should represent an entity or a functionality: if we replace the pattern we use, we will need to rename all these classes.

On the other hand, I realise now that all factory classes contain the word "Factory", which would go in favour of using the word "Composite" here. At least for consistency.

So I'm fine to rollback to "Composite" only for that reason.

Reply:
Maybe we should name if for what it is. `Parsers` or `ParserCollection` or just `Collection`

Reply:
I will go for `Collection` then :)